docs/UserGuide.adoc

```diff
@@ -227,6 +227,35 @@ Format: `exit`
 Address book data are saved in the hard disk automatically after any command that changes the data. +
 There is no need to save manually.
 
+== Features(Coming in v2.0)
+
+=== Add relationship
+
+Enters the indices of the two people in between whom the user wants to add an relationship +
+as seen in the last shown list. +
+Format: `Add Relationship NAME1 NAME2`
```

Comment:
Inconsistency in the description and the command `FORMAT`. Above, you mentioned that a user should enter the "indices of the two people in between whom the user wants to add a relationship", but here your format suggests otherwise. Please make sure that your descriptions align with each other